### DISCUSSION TITLE
关于SEVI大作业的几个问题

### DISCUSSION THREAD

**Discussion by foreverkon** [physics-data/faq#85] [Q&A] [answered]
## 我已仔细阅读了README，但仍然有几个地方不太明白：

1. 没有给出 $R_0$ 和 $\delta_R$的具体数值。
    这个问题似乎已经解决了: [#81 ]
2. 在`Level3`中提到，从微通道板末端出射的电子具有一定的角分布，这个角分布指的是什么？
3. 同样在`Level3`中，CCD像素值是电子数，此时是否不再考虑电子在荧光屏上的亮斑？
4. `第一阶段大作业`的第二种分布中，N(100, 10)中的10是标准差还是方差，还有后面的白噪声N(0,2)也是同样的问题。
5. 在`Level1`和'`Level2`中，需要考虑微通道板的放大作用吗，如果不考虑，那这100个电子打在2048*2048的屏上跟没有一样。

**Comment by Harry-Chen** [comment]
@littlewu2508

**Comment by littlewu2508** [comment, marked as answer]
> ## 我已仔细阅读了README，但仍然有几个地方不太明白：
> 
>     1. 没有给出 $R_0$ 和 $\delta_R$的具体数值。
>        这个问题似乎已经解决了: [[SEVI作业中R_0的值如何确定？ #81](https://github.com/physics-data/faq/discussions/81) ]
> 
>     2. 在`Level3`中提到，从微通道板末端出射的电子具有一定的角分布，这个角分布指的是什么？
> 

这个角分布的定义就写在这句话的后面：cosΘ 的三次方服从0-0.996的均匀分布；轴对称。

>     3. 同样在`Level3`中，CCD像素值是电子数，此时是否不再考虑电子在荧光屏上的亮斑？
不需要；或者说，某区域内轰击到荧光屏的电子束与荧光屏在该区域的亮度成正比，因此统计电子数就可以了。
> 
>     4. `第一阶段大作业`的第二种分布中，N(100, 10)中的10是标准差还是方差，还有后面的白噪声N(0,2)也是同样的问题。
> 
我的锅，这里面所有正态分布我都写成 N(\mu,\sigma)了，所以 N(100,10) 中的 10 是方差。但规范的写法是 N(\mu,\sigma^2)。
>     5. 在`Level1`和'`Level2`中，需要考虑微通道板的放大作用吗，如果不考虑，那这100个电子打在2048*2048的屏上跟没有一样。
微通道板的放大作用，在 level1 与 leve2 中都被建模好了，放大过程+荧光过程+拍照被打包成一个模型——一个入射 MCP 的电子生成一个亮斑。

**Reply by foreverkon** [reply]
谢谢助教。
其他的都明白了，但那个角分布还是不太清楚。
θ和φ是相对哪个坐标系而言的？

**Reply by littlewu2508** [reply]
这个球坐标系的极轴是 MCP 平面的法线，即电子球漂移的方向。

**Reply by foreverkon** [reply]
那为什么没有 `r` 呢？

**Reply by littlewu2508** [reply]
***角分布***

**Reply by foreverkon** [reply]
可是只有θ和φ好像并不能确定电子的位置啊？。。我太憨了
而电子经过MCP后，原先的角分布是不是就没有用了啊

**Reply by littlewu2508** [reply]
这个角分布就是指电子经过一层 MCP 倍增后，出射的许许多多电子的角分布。不存在“入射的角分布”，假设入射的单个电子的入射角对于倍增过程没有影响。

**Reply by foreverkon** [reply]
这`出射的许许多多电子的角分布`也不能确定电子在荧光屏上的位置啊(我问得都不太好意思了，希望助教不用嫌弃我。。

**Reply by heroxbd** [reply]
> 这个角分布就是指电子经过一层 MCP 倍增后，出射的许许多多电子的角分布。不存在“入射的角分布”，假设入射的单个电子的入射角对于倍增过程没有影响。

给一个两层 MCP 叠放的相对位置，电子出射角度，以及屏光幕的位置的示意图。怎么样？

**Reply by littlewu2508** [reply]
![MCPmodel](https://user-images.githubusercontent.com/42602151/131624391-381b3411-ab2b-4f05-a3cd-59fd6a67012f.png)

**Reply by littlewu2508** [reply]
当然，画的有点夸张了，MCP 之间以及与荧光屏的距离 d1 d2 很小，一个电子倍增后只会导致荧光屏很小的区域有电子轰击

**Reply by foreverkon** [reply]
哦，我懂了，谢谢助教！！
原来d1和d2可以用来确定电子位置，我之前一直不知道这两个量有什么用

**Reply by foreverkon** [reply]
这些问题我都清楚了，但在做`基础内容`的时候又遇到了新的问题。
我的理解是这样的: 对于每一个分布，进行50000次试验，每次实验随机取100左右的光电子，统计这些光电子落在CCD上的位置。
但是这样的话，CCD像素有2048*2048个，但光电子数只有100左右，画出来的图什么也没有。
README中有这样一句话：`一次实验要采集 50000 张照片，才可以获得较好的成像`。
这是否意味着每一次实验我实际上要进行50000次采样？？
可是这样的话，最后存储的时候是针对每次实验而不是每次采样，而每次采样得到的结果也不一样，这数据又应该怎么存呢？？

**Reply by littlewu2508** [reply]
每一次采样只需要采集100个电子。确实，绘制出来的图像因为电子数过少而看不见，不过没关系，我会检查代码。

一次实验需要 50000 张采样。每次采样得到的结果也不一样。参见  https://github.com/physics-data/faq/discussions/101 ，50000 确实数据量比较大，需要妥善处理数据结构。
